src/common/common-types.ts

```diff
@@ -134,6 +134,12 @@ export interface ColorInput extends InputBase {
     readonly def: string;
     readonly channels?: readonly number[] | null;
 }
+
+export interface StaticValueInput extends InputBase {
+    readonly kind: 'static';
+    readonly valueOf: 'execution_number' | 'datetime' | 'auto';
```

Comment:
What's `auto`?

Reply:
Hypothetical case where we determine the value based on the schemaId rather than the valueOf type. I could just get rid of this since it's not used anywhere right now, and it's probably not even a good idea to support.

Reply:
Please do. I'm not a fan of not-implemented features.